tests/testapp/tests/helpers.py

```diff
@@ -476,3 +485,12 @@ def setUp(self):
 
     def setUpBeforeMigration(self, apps):
         pass
+
+    @classmethod
+    def tearDownClass(cls):
+        # revert migration back to latest migration
+        executor = MigrationExecutor(connection)
+        executor.loader.build_graph()
+        executor.migrate([cls.latest_migration])
+
+        super(TestMigrationsMixin, cls).tearDownClass()
```

Comment:
@rtibbles This was causing the postgres test failures because it wasn't migrated back to the latest state after the migration tests ran. I can put this change in Kolibri's copy too?